cli/tests/039_worker_deno_ns.ts

```diff
@@ -1,4 +1,7 @@
-const w1 = new Worker("./039_worker_deno_ns/has_ns.ts", { type: "module" });
+const w1 = new Worker("./039_worker_deno_ns/has_ns.ts", {
+  type: "module",
+  deno: true,
+});
```

Comment:
Revert; these files are being removed in #5128; please add a test case to `cli/tests/worker_test.ts`